code/drasil-example/Drasil/GlassBR/Unitals.hs

```diff
@@ -364,9 +364,9 @@ gbConstants = [constant_M, constant_K, constant_ModElas, constant_LoadDur, const
                 ++ gBRSpecParamVals 
 
 constant_M, constant_K, constant_ModElas, constant_LoadDur, constant_LoadDF, constant_LoadSF :: QDefinition
-constant_K       = mkQuantDef sflawParamK  $ (dbl 2.86) * (10 $^ (negate 53))
+constant_K       = mkQuantDef sflawParamK  $ dbl 2.86e-53
```

Comment:
Excellent. Have you searched for all such notation hacks?

Reply:
I checked again. All hacks have been removed.

Reply:
Good, thanks.